core/assets/bundles/bundle_id_ID.properties

```diff
@@ -892,22 +894,22 @@ sector.crevice.name = Retakan
 sector.siege.name = Pengepungan
 sector.crossroads.name = Simpangan
 sector.karst.name = Kars
-sector.origin.name = Origin
+sector.origin.name = Awal Mula
 
 sector.onset.description = Permulaan Penaklukan Erekir. Kumpulkan sumber daya, produksi unit, dan mulailah meneliti teknologi.
 sector.aegis.description = Sektor ini mengandung deposit Tungsten.\nRiset [accent]Bor Tumbukan[] untuk menambang sumber daya ini, dan hancurkan markas musuh di area tersebut.
-sector.lake.description = Danau lava di sektor ini sangat membatasi pilihan unit yang kita gunakan. Unit Kapal adalah satu-satunya pilihan.\nRiset [accent]fabrikator kapal[] dan produksi unit [accent]elude[] secepat mungkin.
-sector.intersect.description = Pemindai menunjukkan bahwa sektor ini akan diserang dari berbagai arah setelah kita mendarat.\nSegera siapkan pertahanan dan perluas wilayah secepat mungkin.\nUnit [accent]mech[] akan dibutuhkan untuk dataran yang sulit dilalui.
-sector.atlas.description = Sektor ini memiliki dataran yang unik dan akan membutuhkan beraneka ragam unit untuk menyerang secara efektif.\nUnit yang ditingkatkan mungkin dibutuhkan untuk melewati beberapa markas musuh yang kuat di sekitar sini.\nRiset [accent]Elektroliser[] dan [accent]Refabrikator Tank[].
+sector.lake.description = Danau lava di sektor ini sangat membatasi pilihan unit yang kita gunakan. Unit Kapal adalah satu-satunya pilihan.\nRiset [accent]Pabrikator Kapal[] dan produksi unit [accent]Elude[] secepat mungkin.
```

Comment:
Elude is okay but why the Ship Fabricator must be capitalized? thats Noun(nomina).
[EYD Guide of Capitalization](https://ejaan.kemdikbud.go.id/eyd/penggunaan-huruf/huruf-kapital/#10-huruf-kapital-digunakan-sebagai-huruf-pertama-unsur-nama-jabatan-dan-pangkat-yang-diikuti-nama-orang-atau-yang-digunakan-sebagai-pengganti-nama-orang-nama-instansi-atau-nama-tempat)
[PUEBI Guide of Capitalization (3rd Party)](https://ivanlanin.github.io/puebi/huruf/huruf-kapital/)